### PR TITLE
Cleanups 20010702

### PR DESCRIPTION
**Pull request by pcfe**
- a better description in Satellite for the builds (build number at start of line and info on Jenkins jobshown
- a coupe minor changes that make reading logs more pleasant.

Please merge.

### CODE REVIEW

File: publishcv.sh [hunk 1]
```diff
@@ -32,21 +32,25 @@ IFS="${oldIFS}"
...
 
     # get the latest version of each CV, add it to the array
+    info "Get the latest version of CV ${cv}"
     VER_ID_LIST+=( "$(ssh -q -l ${PUSH_USER} -i ${RSA_ID} ${SATELLITE} \
 	"hammer content-view info --name \"${cv}\" --organization \"${ORG}\" \
 	| sed -n \"/Versions:/,/Components:/p\" | grep \"ID:\" | tr -d ' ' | cut -f2 -d ':' | sort -n | tail -n 1")" )
 done
 
 # sleep after publishing content view to give chance for locks to get cleared up
+info "Give Satelllite 90 seconds to settle WRT Content View locks"
```
Comment (evgeni) [inline publishcv.sh]: Satellite is written with 2 L, not 3.

File: publishcv.sh [hunk 2]
```diff
@@ -95,16 +99,19 @@ then # there is at least one CCV using the given CVs
             cv_used_ver_ids=${CV_USED_VER_IDS[$i]}
 
             # we add back the CV under its latest version to the CCV
+            info "Update component IDs of CCV ${ccv_id}"
 	    ssh -q -l ${PUSH_USER} -i ${RSA_ID} ${SATELLITE} \
                 "hammer content-view update --id ${ccv_id} --component-ids \"${cv_used_ver_ids}\" --organization \"${ORG}\"" || \
 	            { err "CCV '${ccv_id}' couldn't be updated with '${cv_used_ver_ids}'."; exit 1; }
 
             # sleep after updating CV for locks to get cleared up
+            info "Give Satelllite 10 seconds to settle WRT Content View locks"
```
Comment (evgeni) [inline publishcv.sh]: LLL

File: publishcv.sh (hunk 1 above)
Comment (pcfe) [inline publishcv.sh]: thx, fixed with 437b525

File: publishcv.sh (hunk 2 above)
Comment (pcfe) [inline publishcv.sh]: also fixed with 437b525

**Pull request merged**